src/modules/dayCycle/DayCycle.ts

```diff
@@ -2,18 +2,19 @@
 import { Computed } from 'knockout';
 import DayCycleMoment from './DayCycleMoment';
 import DayCyclePart from './DayCyclePart';
+import { DayCycleStartHours } from '../GameConstants';
 import GameHelper from '../GameHelper';
 
 export default class DayCycle {
     public static currentDayCyclePart: Computed<DayCyclePart> = ko.pureComputed(() => {
         const curHour = GameHelper.currentTime().getHours();
         let dayCyclePart: DayCyclePart;
 
-        if (curHour === 17) {
+        if (curHour === DayCycleStartHours.dusk) {
             dayCyclePart = DayCyclePart.Dusk;
-        } else if (curHour === 6) {
+        } else if (curHour === DayCycleStartHours.dawn) {
             dayCyclePart = DayCyclePart.Dawn;
-        } else if (curHour > 6 && curHour < 17) {
+        } else if (curHour > DayCycleStartHours.dawn && curHour < DayCycleStartHours.dusk) {
```

Comment:
When you have filled in all DayCycleStartHours, this if/else can be replaced with finding the smallest DayCycleStartHour which is greater than the current hour, or the last DayCyclePart if none was found.